### PR TITLE
Add Autobahn testing to the CI

### PR DESCRIPTION
**Pull request by pushkarnk**

### CODE REVIEW

**Comment by pushkarnk** [comment]
The CI will now fail in three scenarios:
1. If all of the configured autobahn tests don't run
2. If autobahn ends up in a hang
3. If there are failures or unclean closures

File: .autobahn.sh [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    # Generate the fuzzingclient.json
+    fuzzing_client $TESTS
+
+    # Run the autobahn fuzzingclient
+    wstest -m fuzzingclient
+
+    # Count the number of failed tests or unclean connection closures
+    FAILED_OR_UNCLEAN=`grep behavior reports/servers/index.json | cut -d':' -f2 | cut -d'"' -f2 | sort -u | xargs | grep -E "FAILED|UNCLEAN" | wc -l`
+    if [ $FAILED_OR_UNCLEAN -ne "0" ]; then
+        return $FAILED_OR_UNCLEAN
```
Comment (djones6) [inline .autobahn.sh]: Would it be worth printing a message explaining why we are failing (and/or how many failures there were), or would this be evident already from wstest's output?

File: .autobahn.sh [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+swift test
+SWIFT_TEST_STATUS=$?
+travis_end
+if [ $SWIFT_TEST_STATUS -ne 0 ]; then
+    return $SWIFT_TEST_STATUS
+fi
+
+# Build TestWebSocketService
+echo "Building in release mode for autobahn testing"
+swift build -c release
```
Comment (djones6) [inline .autobahn.sh]: I think this part should also be wrapped in a travis_start / travis_end, as it's quite a sizeable chunk of output (particularly with BoringSSL).

**Comment by djones6** [comment]
@pushkarnk The CI is failing because of a problem with the swift:5.0.1 docker images.  I filed a bug for it here: https://bugs.swift.org/browse/SR-10591
I have a PR that works around this against Package-Builder which we can use here (https://github.com/IBM-Swift/Package-Builder/pull/166)

**Pull request merged**